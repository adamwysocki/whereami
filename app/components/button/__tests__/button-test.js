// __tests__/button-test.js
import 'react-native';
import React from 'react';
import Button from '../index';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

var onPress = function() {
  return 1;
}

describe('Button', () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <Button onPress={onPress}>PRESS</Button>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
