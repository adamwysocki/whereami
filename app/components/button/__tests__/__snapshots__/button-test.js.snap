exports[`Button renders correctly 1`] = `
<div
  className="TouchableHighlight"
  onClick={[Function onPress]}
  onPress={[Function onPress]}>
  <div
    className="View"
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0000FF",
        "borderRadius": 5,
        "height": 50,
        "justifyContent": "center",
        "width": 250
      }
    }>
    <div
      className="Text"
      style={
        Object {
          "color": "white",
          "fontWeight": "900",
          "textAlign": "center"
        }
      }>
      PRESS
    </div>
  </div>
</div>
`;
